### PR TITLE
Handle original file missing and destination file existing.

### PR DESCRIPTION
**Pull request by tokejepsen**

### CODE REVIEW

File: pype/plugins/global/publish/integrate_new.py [hunk 1]
```diff
@@ -952,21 +954,37 @@ def handle_destination_files(self, integrated_file_sizes, mode):
...
-
-                except FileNotFoundError:
-                    pass  # file not there, nothing to delete
+                        new_name = re.sub(
+                            r'\.{}$'.format(self.TMP_FILE_EXT),
+                            '',
+                            file_url
+                        )
+
+                        if os.path.exists(new_name):
```
Comment (iLLiCiTiT) [inline pype/plugins/global/publish/integrate_new.py:974]: Is possible that `new_name == file_url`? In that case I would skip.

Comment (tokejepsen) [reply pype/plugins/global/publish/integrate_new.py:974]: No, `new_name` is `file_url + self.TMP_FILE_EXT`.

**Review by iLLiCiTiT — APPROVED** [review]
Seems good to me.

**Review by kalisp — APPROVED** [review]

**Comment by jezscha** [comment]
I wonder, is this bugfix needed in `2.x/develop` or in `release/2.12.0-rc`?

**Comment by iLLiCiTiT** [comment]
> I wonder, is this bugfix needed in 2.x/develop or in release/2.12.0-rc?

I would say that it's for release

**Comment by jezscha** [comment]
> > I wonder, is this bugfix needed in 2.x/develop or in release/2.12.0-rc?
> 
> I would say that it's for release

would you be able to redo this from release branch? `release/2.12.0-rc`. When I change the target it want work.

**Pull request closed without merging**